lib/services.rb

```diff
@@ -12,6 +13,8 @@ def self.content_store
   end
 
   def self.email_sender
-    @email_sender ||= EmailSender::Notify.new
+    return @email_sender ||= EmailSender::Notify.new if EmailAlertAPI.config.email_service_provider == "NOTIFY"
+    return @email_sender ||= EmailSender::Pseudo.new if EmailAlertAPI.config.email_service_provider == "PSEUDO" || EmailAlertAPI.config.email_service_provider.nil?
+    raise NameError, "Email service provider does not exist"
```

Comment:
Wonder if it's worth including the email service provider that was given in the error message?

Reply:
Also not sure if `NameError` is the right exception to use here as it feels like it has a very specific use case of undefined variables? But maybe it's normal to use this for other things in Ruby.

I would suggest an ordinary `RuntimeError`.

Reply:
I liked NameError since it's "Raised when a given name is invalid or undefined." which is basically what we have, albeit a class that is literally undefined/non-existent. Anyhoo, bog standard generic raise now in place.